brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
I think I'd be fine with it if the method started with `has_` to match `has_dwolla_payments`. That would be the start of a convention / something we could see and know needed to be fixed?